DoctorWho.Domain/EpisodeCompanion.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace DoctorWho.Domain
+{
+    public class EpisodeCompanion
+    {
+        public int EpisodeCompanionId { get; set; } 
+        public int EpisodeId { get; set; } 
+        public int CompanionId { get; set; }
+
```

Comment:
you can use EpisodeId and Campanion Id as composite primary key, no need for  EpisodeCompanionId